lob-java-examples/src/main/java/com/lob/examples/CsvLetterExample.java

```diff
@@ -0,0 +1,103 @@
+package com.lob.examples;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
+import com.fasterxml.jackson.databind.MappingIterator;
+import com.fasterxml.jackson.dataformat.csv.CsvMapper;
+import com.fasterxml.jackson.dataformat.csv.CsvSchema;
+import com.google.common.base.Charsets;
+import com.google.common.io.Files;
+import com.lob.ClientUtil;
+import com.lob.client.AsyncLobClient;
+import com.lob.client.LobClient;
+import com.lob.protocol.request.AddressRequest;
+import com.lob.protocol.request.LetterRequest;
+import com.lob.protocol.response.LetterResponse;
+
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.HashMap;
+import java.util.Map;
+
+public class CsvLetterExample extends BaseExample {
+    public static void main(final String[] args) throws Exception {
+        final LobClient client = AsyncLobClient.createDefault(API_KEY);
+
+        final String letterTemplate = Files.toString(ClientUtil.fileFromResource("letters/letter_template.html"), Charsets.UTF_8);
+
+        final AddressRequest companyAddress = AddressRequest.builder()
+                .name("Deluxe Virgina Realty")
+                .line1("185 Berry St.")
+                .line2("Ste 6600")
+                .city("San Francisco")
+                .state("CA")
+                .zip("94107")
+                .country("US")
+                .build();
+
+        final String today = new SimpleDateFormat("MMMM d YYYY").format(new Date());
+
+        final CsvMapper mapper = new CsvMapper();
+
+        final CsvSchema schema = mapper.schemaFor(CsvLetter.class).withHeader();
+        final MappingIterator<CsvLetter> iterator = mapper.reader(CsvLetter.class).with(schema).readValues(fileFromResource("letters/input.csv"));
+        while (iterator.hasNext()) {
+            final CsvLetter letter = iterator.next();
+
+            final Map<String, String> data = new HashMap<String, String>();
+            data.put("date", today);
+            data.put("name", letter.name);
+            data.put("amountDue", letter.amount);
+
+            final LetterRequest letterRequest = LetterRequest.builder()
+                    .to(
+                            AddressRequest.builder()
```

Comment:
There is no standard convention for line-wrapping but it's standard to increase the indentation space for a new block. Although this isn't a new block, it is all supposed to be enclosed in the `.to()`. For what it's worth, it's the convention in the JUnit tests.

https://github.com/lob/lob-java/blob/9cdeb6d15066d7faaea9d6eeaf57635d601ddeb2/src/test/java/com/lob/client/test/PostcardTest.java#L109